### PR TITLE
Add more validation

### PR DESCRIPTION
**Pull request by zvs001**
added validation for initialize. It will help to avoid crashes.

Possible fix for: https://github.com/mieszko4/react-native-zoom-us/issues/44

### CODE REVIEW

File: index.ts [hunk 1]
```diff
@@ -21,12 +22,13 @@ async function initialize(
...
-        'Check Link: https://github.com/mieszko4/react-native-zoom-us/blob/master/docs/UPGRADING.md',
-    )
-  }
+  invariant(typeof params === 'object',
+    'ZoomUs.initialize expects object param. Consider to check migration docs. ' +
+    'Check Link: https://github.com/mieszko4/react-native-zoom-us/blob/master/docs/UPGRADING.md',
+  )
+
+  invariant(params.clientKey, 'ZoomUs.initialize requires clientKey')
+  invariant(params.clientSecret, 'ZoomUs.initialize requires clientSecret')
```
Comment (mieszko4) [inline index.ts:31]: :+1:

**Review by mieszko4 — COMMENTED** [review]
thnx, it is good but it depends on https://github.com/mieszko4/react-native-zoom-us/pull/46 which needs to be resolved/merged first.
So please let's resolve https://github.com/mieszko4/react-native-zoom-us/pull/46 first then  we can merge this one.
(Or base your invariant lib changes on current master)

**Comment by zvs001** [comment]
Yes. Just wait for previous one. 
I didn't want to wait or merge conflicts, so created like this.

**Comment by mieszko4** [comment]
@zvs001 please pull master now, so we can merge this one, too

**Pull request merged**